sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -102,81 +104,68 @@ export interface Response {
 }
 
 /**
- * Represents properties of the namespace.
+ * @interface
+ * An interface that tracks the settings for paged iteration
  */
-export interface NamespacePropertiesResponse extends NamespaceProperties, Response {}
+export interface PageSettings {
```

Comment:
Why not just re-export this from `@azure/core-paging`? It doesn't seem to make sense to duplicate it.

Reply:
`continuationToken` is a "number" here and a "string" in the PageSettings from core-paging, so I guess a new interface is needed.

Reply:
Or should core-paging change?

Reply:
Ah I didn't notice that. Using a number is interesting; I think many continuationTokens are opaque strings. 

I'm fine having a custom PageSettings, or you could always stringify and parseInt the number if you wanted to use the string form of the interface.

Reply:
Service returns the "number" as a string.

"number" makes more sense for the "continuationToken" here because it is basically the number of items to skip from the top. Users can provide an arbitrary number and start listing from there and no need to reach out to the service for a continuationToken.